modules/imgproc/perf/perf_integral.cpp

```diff
@@ -5,9 +5,22 @@
 
 namespace opencv_test {
 
+enum PerfSqMatDepth{
+    CV_32S32S = 0,
+    CV_32S32F,
+    CV_32S64F,
+    CV_32F32F,
+    CV_32F64F,
+    CV_64F64F};
+
+CV_ENUM(SqSizes, CV_32S32S, CV_32S32F, CV_32S64F, CV_32F32F, CV_32F64F, CV_64F64F);
```

Comment:
> SqSizes

This doesn't look as "sizes". Perhaps it should be `IntergralOutputDepths`

Also it is better to replace `CV_` prefix to avoid confusion with public OpenCV API.